scripts/index.js

```diff
@@ -0,0 +1,74 @@
+function getRandomInt(min, max) {
+	min = Math.floor(min);
+	max = Math.ceil(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+
+let diceOne = {
+	value: 1,
+	roll : function(){
+		this.value = getRandomInt(1, 6);
+		return this.value;
```

Comment:
Well done, it is considered good practice to return a value even from object methods which mutate the object.